setup.py

```diff
@@ -230,6 +230,7 @@
             'zenodo_rest = zenodo.modules.rest.views:blueprint',
             'zenodo_deposit = zenodo.modules.deposit.views_rest:blueprint',
             'zenodo_spam = zenodo.modules.spam.views:blueprint',
+            'zenodo_metrics = zenodo.modules.metrics.views:blueprint',
```

Comment:
Also add under `invenio_base.api_apps` an entry for the extension, i.e. `zenodo.modules.metrics.ext:ZenodoMetrics`